py/picca/test/test_helpers.py

```diff
@@ -0,0 +1,201 @@
+import configparser as ConfigParser
+from picca.utils import userprint
+import fitsio
+import os
+import numpy as np
+import h5py
+import sys
+
+
+def update_system_status_values(path, section, system, value):
+
+    ### Make ConfigParser case sensitive
+    class CaseConfigParser(ConfigParser.ConfigParser):
+
+        def optionxform(self, optionstr):
```

Comment:
Why is this a function and not a variable?

Reply:
the whole purpose of this class is to redefine the function from ConfigParser which else would be 
```def optionxform(self, optionstr):
        return optionstr.lower()```
and thus would not be case sensitive.

Reply:
should we keep it then?

Reply:
yes, else the configfile will not be properly modified ;-)